### PR TITLE
Show applied filters in browse events.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR shows shows the applied filter when no results are found on browse events page. 

#### Changes proposed in this pull request:

- Show the filters applied ~~when no results are obtained from the filter~~ in the explore events.

## Screenshot
Screenshot from 2019-04-10 10-58-46

Screenshot from 2019-04-10 10-58-27

Fixes: #2548

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
Thanks for the suggestion. I was looking for something like that only. Just for an enhancement,can we show these in results also I.e when there are filtered events?

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have made the requested changes. I have updated the PR description also.

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have implemented the required changes.

File: app/templates/explore.hbs [hunk 1]
```diff
@@ -4,6 +4,38 @@
...
     <h1 class="ui header">{{t 'Events'}}</h1>
+    <div class="ui labels">
+      {{#if filters.category}}
+        <a class="ui mini label">
+          {{filters.category}}
+          <a role="button" {{action 'clearFilter' 'category'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.sub_category}}
+        <a class="ui mini label">
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: div

File: app/templates/explore.hbs [hunk 2]
```diff
@@ -4,6 +4,38 @@
...
+          {{filters.sub_category}}
+          <a role="button" {{action 'clearFilter' 'sub_category'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.event_type}}
+        <a class="ui mini label">
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: div

File: app/templates/explore.hbs [hunk 3]
```diff
@@ -4,6 +4,38 @@
...
+          {{filters.event_type}}
+          <a role="button" {{action 'clearFilter' 'event_type'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.start_date}}
+        <a class="ui mini label">
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: div

File: app/templates/explore.hbs [hunk 4]
```diff
@@ -4,6 +4,38 @@
...
+          {{moment-format filters.start_date 'ddd, MMM DD'}}
+          <a role="button" {{action 'clearFilter' 'start_date'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.end_date}}
+        <a class="ui mini label">
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: div

File: app/templates/explore.hbs [hunk 5]
```diff
@@ -4,6 +4,38 @@
...
+          {{moment-format filters.end_date 'ddd, MMM DD'}}
+          <a role="button" {{action 'clearFilter' 'end_date'}}><i class="icon close"></i></a>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: why wrap it inside an extra anchor tag?

File: app/templates/explore.hbs [hunk 6]
```diff
@@ -4,6 +4,38 @@
...
+      {{#if filters.event_type}}
+        <a class="ui mini label">
+          {{filters.event_type}}
+          <a role="button" {{action 'clearFilter' 'event_type'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.start_date}}
+        <a class="ui mini label">
+          {{moment-format filters.start_date 'ddd, MMM DD'}}
+          <a role="button" {{action 'clearFilter' 'start_date'}}><i class="icon close"></i></a>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: why wrap it inside an extra anchor tag?

File: app/templates/explore.hbs [hunk 7]
```diff
@@ -4,6 +4,38 @@
...
+      {{#if filters.category}}
+        <a class="ui mini label">
+          {{filters.category}}
+          <a role="button" {{action 'clearFilter' 'category'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.sub_category}}
+        <a class="ui mini label">
+          {{filters.sub_category}}
+          <a role="button" {{action 'clearFilter' 'sub_category'}}><i class="icon close"></i></a>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: why wrap it inside an extra anchor tag?

File: app/templates/explore.hbs [hunk 8]
```diff
@@ -4,6 +4,38 @@
...
+        </a>
+      {{/if}}
+      {{#if filters.event_type}}
+        <a class="ui mini label">
+          {{filters.event_type}}
+          <a role="button" {{action 'clearFilter' 'event_type'}}><i class="icon close"></i></a>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: why wrap it inside an extra anchor tag?

File: app/templates/explore.hbs [hunk 9]
```diff
@@ -4,6 +4,38 @@
   </div>
   <div class="twelve wide column">
     <h1 class="ui header">{{t 'Events'}}</h1>
+    <div class="ui labels">
+      {{#if filters.category}}
+        <a class="ui mini label">
+          {{filters.category}}
+          <a role="button" {{action 'clearFilter' 'category'}}><i class="icon close"></i></a>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: why wrap it inside an extra anchor tag?

File: app/templates/explore.hbs [hunk 10]
```diff
@@ -4,6 +4,38 @@
   </div>
   <div class="twelve wide column">
     <h1 class="ui header">{{t 'Events'}}</h1>
+    <div class="ui labels">
+      {{#if filters.category}}
+        <a class="ui mini label">
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: div

File: app/templates/explore.hbs (hunk 5 above)
Comment (Anupam-dagar) [reply app/templates/explore.hbs]: I can do tag i with role=button but that will remove the hover effect currently coming due to 'a' tag. Changing i to a leads to css issue of the icon.

Comment (CosmicCoder96) [reply app/templates/explore.hbs]: okay cool, leave it, do make the labels as div instead of a.

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have changed all the `a` tags to `div`.

**Pull request merged**